config/config.go

```diff
@@ -81,15 +81,21 @@ type Fmt struct {
 
 // Plan is a terraform plan config
 type Plan struct {
-	Template    string      `yaml:"template"`
-	WhenDestroy WhenDestroy `yaml:"when_destroy,omitempty"`
+	Template      string        `yaml:"template"`
+	WhenDestroy   WhenDestroy   `yaml:"when_destroy,omitempty"`
+	WhenNoChanges WhenNoChanges `yaml:"when_no_changes,omitempty"`
 }
 
 // WhenDestroy is a configuration to notify the plan result contains destroy operation
 type WhenDestroy struct {
 	Template string `yaml:"template"`
```

Comment:
I think it's better to have the label when destroying as well to make it more visible.

Reply:
I can agree but perhaps it's out of scope for this PR?